tests/main/op-remove/task.yaml

```diff
@@ -1,5 +1,6 @@
 summary: Check snap remove operations.
-
+# we're using test-snapd-busybox-static which is published for amd64 only
+systems: [-ubuntu-16.04-32]
```

Comment:
Let me fix test-snapd-busybox-static instead, this will cause trouble down the road when we run this test on arm etc